### PR TITLE
add captcha to registration form

### PR DESCRIPTION
**Pull request by goapunk**
might need styling

### CODE REVIEW

**Comment by goapunk** [comment]
WIP because it depends on #2408

**Comment by goapunk** [comment]
needs https://github.com/liqd/admin/pull/362 to be merged with it

**Review by fuzzylogic2000 — COMMENTED** [review]
It cannot find the js:
```
GET http://localhost:8001/static/captcheck.js 
net::ERR_ABORTED 404 (Not Found)
```

**Comment by fuzzylogic2000** [comment]
> It cannot find the js:
> 
> ```
> GET http://localhost:8001/static/captcheck.js 
> net::ERR_ABORTED 404 (Not Found)
> ```

Right, I added it to webpack. Will push that!
But it is still not totally happy with the js:
```
captcheck.js:31 Uncaught TypeError: Cannot set properties of undefined (setting 'captcheck')
    at captcheck.js:31
    at captcheck.js:33
(anonymous) @ captcheck.js:31
(anonymous) @ captcheck.js:33

```
But maybe happy enough?

**Comment by goapunk** [comment]
> It cannot find the js:
> 
> ```
> GET http://localhost:8001/static/captcheck.js 
> net::ERR_ABORTED 404 (Not Found)
> ```

Ah, the entry point in webpack got lost, will add it

**Comment by fuzzylogic2000** [comment]
> > It cannot find the js:
> > ```
> > GET http://localhost:8001/static/captcheck.js 
> > net::ERR_ABORTED 404 (Not Found)
> > ```
> 
> Ah, the entry point in webpack got lost, will add it

Sorry, already did

File: webpack.common.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ module.exports = {
       './euth_wagtail/assets/scss/all.scss',
       './euth_wagtail/assets/js/app.js'
     ],
+    captcheck: [
+      './euth/captcha/assets/captcheck.js'
+    ],
```
Comment (goapunk) [inline webpack.common.js:37]: should this have the `dependOn: 'adhocracy4'` as the others?

Comment (fuzzylogic2000) [reply webpack.common.js:37]: Yes!

**Review by fuzzylogic2000 — APPROVED** [review]
Yeah!

**Pull request merged**